parser/parser.y

```diff
@@ -2176,13 +2177,17 @@ Function:
 |	FunctionCallAgg
 
 FunctionNameConflict:
-	"DATABASE" | "SCHEMA" | "IF" | "LEFT" | "REPEAT" | "CURRENT_USER" | "CURRENT_DATE" | "VERSION" | "UTC_DATE"
+	"ASCII" | "DATABASE" | "SCHEMA" | "IF" | "LEFT" | "REPEAT" | "CURRENT_USER" | "CURRENT_DATE" | "VERSION" | "UTC_DATE"
```

Comment:
ASCII is not a reserved keyword. It should not be here. You can put it into UnReservedKeyword rule: https://github.com/pingcap/tidb/blob/master/parser/parser.y#L1819

See: https://dev.mysql.com/doc/refman/5.7/en/keywords.html